README.md

```diff
@@ -172,7 +172,7 @@ Create a messaging session in your client application to receive messages from A
 
 ```js
 import * as AWS from 'aws-sdk/global';
-import * as Chime from 'aws-sdk/clients/chime';
+import Chime from 'aws-sdk/clients/chime';
```

Comment:
I think we should test before making this change, I had check with @kmcqueen-amzn who ran into issue when running the demo chat app after using the import without wildcard import.

Reply:
Oh, I thought we just need to update it as you commented in that issue. 
So @kmcqueen-amzn ran into issue without wildcard import while the builder has issue with the wildcard import?

Reply:
Yup. Sorry my bad I commented that it was a documentation update as I saw a commit for the same from @kmcqueen-amzn in the issue. Can you please sync with him once and check if you run into the same issue?

Reply:
I saw he made a commit in his branch, but never create a PR. 
Sure, will reach out to him.

Reply:
Thanks for the reminder Venky! just confirmed with @kmcqueen-amz, wildcard import works with the demo as expected, but removing the wildcard *does not* work.

Will close this PR.